dcrlibwallet.go

```diff
@@ -112,13 +113,9 @@ func newLibWallet(walletDataDir, walletDbDriver string, activeNet *netparams.Par
 func (lw *LibWallet) Shutdown() {
 	log.Info("Shutting down dcrlibwallet")
 
-	// Trigger shuttingDown signal to cancel all contexts created with `contextWithShutdownCancel`.
+	// Trigger shuttingDown signal to cancel all contexts created with `lw.shutdownContext()`.
 	lw.shuttingDown <- true
 
-	if lw.rpcClient != nil {
-		lw.rpcClient.Stop()
-	}
-
 	lw.CancelSync()
```

Comment:
Should at least wrap this in `if lw.CancelSync != nil`. But as recommended in a previous review, this nil check and `cancelSync()` execution should be done in the go routine that listens for `lw.shuttingDown` signals.

The idea is to cancel all created shutdown contexts in that go routine, and also cancel sync if cancelSync != nil.